generate/beat/{{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go

```diff
@@ -13,42 +13,29 @@ import (
 )
 
 type {{cookiecutter.beat|capitalize}} struct {
-	config config.Config
-	done   chan struct{}
-	client publisher.Client
+	done       chan struct{}
+	config     config.Config
```

Comment:
What I have in mind is running `go fmt` via cookiecutter post-generate hook: http://cookiecutter.readthedocs.io/en/latest/advanced/hooks.html